src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -387,6 +389,40 @@ public void testDataSourceOktaSerialization() throws Exception {
     con.close();
   }
 
+  @Test
+  @Ignore
+  public void testDataSourceOktaGenerates429StatusCode() throws Exception {
```

Comment:
lets move this test to ConnectionLatestIT with comment that it works in version above 3.15.1